src/test/fuzz/versionbits.cpp

```diff
@@ -199,7 +199,7 @@ FUZZ_TARGET_INIT(versionbits, initialize)
     const uint32_t signalling_mask = fuzzed_data_provider.ConsumeIntegral<uint32_t>();
 
     // mine prior periods
-    while (fuzzed_data_provider.remaining_bytes() > 0) {
+    LIMITED_WHILE(fuzzed_data_provider.remaining_bytes() > 0, 10000) {
```

Comment:
Why is this needed? There is already a `break` below, no?

Reply:
Yep, you're right, I misread the loop's exit condition.